src/config-import.js

```diff
@@ -17,24 +18,34 @@ const MAX_ICON_SIZE = 10_000_000
  *   fields:  { [id: string]: unknown }
  * }} PresetsFile
  */
+
+/** @typedef {('presets' | 'fields')} ValidDocTypes */
+/**
+ * @typedef {{
+ *  [lang: string]: unknown
+ * }} TranslationsFile
+ */
+
 /**
  * @typedef {Parameters<import('./icon-api.js').IconApi['create']>[0]} IconData
  */
 
+/** @type {Error[]} */
+let warnings = []
+
 /**
  * @param {string} configPath
  */
 export async function readConfig(configPath) {
-  /** @type {Error[]} */
-  const warnings = []
-
+  warnings = []
   const zip = await yauzl.open(configPath)
   if (zip.entryCount > MAX_ENTRIES) {
     // MAX_ENTRIES in MAC can be inacurrate
     throw new Error(`Zip file contains too many entries. Max is ${MAX_ENTRIES}`)
   }
   const entries = await zip.readEntries(MAX_ENTRIES)
   const presetsFile = await findPresetsFile(entries)
+  const translationsFile = await findTranslationsFile(entries)
```

Comment:
Minor: could we do these concurrently?

```suggestion
  const [presetsFile, translationsFile] = await Promise.all([
    findPresetsFile(entries),
    findTranslationsFile(entries),
  ])
```